### DISCUSSION TITLE
Zustand website uses a lot of CPU

### DISCUSSION THREAD

**Discussion by FarzadMohtasham** [pmndrs/zustand#2635] [General] [answered]
I'm using Zustand in my project, and I am so happy for that, But Main page of Zustand website uses a lot of CPU, Please check it, Thanks

![image_2024-07-07_18-56-04](https://github.com/pmndrs/zustand/assets/48632860/4abdcc8c-768b-4e73-8fdd-22ce5b7afab1)

**Comment by dai-shi** [comment]
We've received a similar issue before, #2249 and maybe some others. Does it look similar or different?

**Reply by FarzadMohtasham** [reply]
Graphic is ok, But it hangs so much and uses a lot of CPU

**Comment by dai-shi** [comment]
Can anyone else reproduce it? Does anyone know how that could happen?

**Reply by dbritto-dev** [reply]
@dai-shi works well for me. I believe is something related to three.js.

**Reply by FarzadMohtasham** [reply, marked as answer]
I tested Zustand website in Mozilla Firefox and Microsoft Edge browsers, It works very well, I think problem is from Google Chrome!
